rfc/0015-multi-file-jayvee/0015-multi-file-jayvee.md

```diff
@@ -65,120 +100,100 @@ valuetype MyValueType2 {
 
 ```
 use * from './path/to/location.jv'; // use all published elements from the file, access via qualified name as if it would be defined at the root level
 use { MyDomainSpecificValuetype1 } from './path/to/location.jv'; // only use the published elements from the file, access via qualified name as if it would be defined at the root level
-use { MyDomainSpecificValuetype1 called Vt1} from './path/to/location.jv'; // only use the published elements from the file, access via qualified name using the alias
+use { MyDomainSpecificValuetype1 as Vt1} from './path/to/location.jv'; // only use the published elements from the file, access via qualified name using the alias
 ```
 
 References to these used elements is by their qualified name (unless altered by an alias).
 
-#### Using a library (from outside of the project)
+#### Using a package
 
 The `use` syntax is similar to importing an element of a file.
-Libraries can only be used as a whole.
+Packages can only be used as a whole.
 
 ```
-use { MyDomainLibrary } from './path/to/location.jv'; // only use the named library, access via qualified name
+use { MyDomainPackage } from './path/to/location.jv'; // only use the named package, access via qualified name
 use {
-  MyDomainLibrary1,
-  MyDomainLibrary1
-} from './path/to/location.jv'; // only use the named libraries, access via qualified name
-use { MyDomainLibrary called MyLibraryAlias} from './path/to/location.jv'; // only use the named library, access via qualified name using the alias
+  MyDomainPackage1,
+  MyDomainPackage2
+} from './path/to/location.jv'; // only use the named packages, access via qualified name
+use { MyDomainPackage as MyPackageAlias} from './path/to/location.jv'; // only use the named package, access via qualified name using the alias
 ```
 
 References to these used elements is by their qualified name (unless altered by an alias).
 
+## Decision rationale
+
+- keywords `publish` and `use` over `export` and `import` since they are less technical and better understandable for subject-matter experts
+- concept of "namespaces" is separate from publishing them for reuse in other projects (this RFC only introduces the concept of "namespaces" by the keyword `package`)
+
 ## Drawbacks
 
-- Two different sharing mechanisms (`publish` keyword, `library` concept`)
-- Elements of a pipeline cannot be reused, leading to potentially more slim pipelines and a parallel library
-- The elements of a library within a file always need the qualified name (alternative: allow access via sole name within file?)
-- The RFC does not allow re-publishing (only by putting elements into a containing library)
+- Two different sharing mechanisms (`publish` keyword, `package` concept`)
+- Elements of a pipeline cannot be reused, leading to potentially more slim pipelines and a parallel package
+- The elements of a package within a file always need the qualified name (alternative: allow access via sole name within file?)
```

Comment:
This is really hard to read (and I don't think `sole name` is an english phrase). But I'd go with allowing elements inside a package to refer to other elements in the same package by their name. But otherwise I think it is okay to have to use the fully qualified name at file level.